src/Foundatio.Redis/Cache/RedisCacheClientOptions.cs

```diff
@@ -11,6 +11,11 @@ public class RedisCacheClientOptions : SharedOptions {
         /// </summary>
         public bool ShouldThrowOnSerializationError { get; set; } = true;
 
+        /// <summary>
+        /// The behaviour required when performing read operations from cache
+        /// </summary>
+        public CommandFlags ReadMode { get; set; } = CommandFlags.None;
```

Comment:
Might want to include a comment on when you would prefer replica vs primary. I'd think you'd still want to prefer replica by default no? @ejsmith 

Also seeing the latest redis client release, wonder if this fixes some of the issues you saw.. https://stackexchange.github.io/StackExchange.Redis/ReleaseNotes

Fix [#2593](https://github.com/StackExchange/StackExchange.Redis/issues/2593): EXPIRETIME and PEXPIRETIME miscategorized as PrimaryOnly commands causing them to fail when issued against a read-only replica ([#2593 by slorello89](https://github.com/StackExchange/StackExchange.Redis/pull/2593))

On a side note, can you please sign the CLA.

Reply:
No, I think you should opt into reading potentially stale data. Really you'd want to make that decision on a case by case basis.

Reply:
@niemyjski I signed the CLA but it still shows that the check fails. maybe I messed something up?
I think I didn't set author when I pushed the commit so maybe it used different user which I use for my other git server? (I imagined using my github key is enough for it to be linked to this account? should I try and create new commit and PR or maybe I'm missing something simpler?)
Thanks.